extensions/adobe/experience/decisioning/proposition.schema.json

```diff
@@ -0,0 +1,96 @@
+          "$ref": "https://ns.adobe.com/xdm/data/record"
+        },
+        "xdm:selections": {
+          "title": "Selected Options",
+          "description": "Snapshots of the options that were selected for this activity. Those options are ususally constrained. The objects contain a at least the @id property value which is the URI of an option. See schema https://ns.adobe.com/experience/decisioning/option",
+          "type": "array",
+          "items": {
+            "$ref": "https://ns.adobe.com/xdm/data/record"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "xdm:fallback": {
+          "title": "Fallback Option",
+          "description": "Snapshot of the fixed fallback option that was selected because no available option satisfied all constaints. The object contains at least the @id property value which is the URI of the fallback action. See schema https://ns.adobe.com/experience/decisioning/fallback-option",
+          "$ref": "https://ns.adobe.com/xdm/data/record"
+        }
+      }
+    },
+    "proposition-metrics-datatype": {
+      "properties": {
+        "xdm:metrics": {
+          "title": "Metrics",
+          "description": "Additional measurements observed while the decision was made and which provided context to the decision process.",
+          "$ref": "https://ns.adobe.com/experience/decisioning/eventmetrics"
+        }
+      }
+    },
+    "propositions-mixin": {
+      "meta:intendedToExtend": [
+        "https://ns.adobe.com/xdm/context/experienceevent",
+        "https://ns.adobe.com/experience/decisioning/decisionevent"
+      ],
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositionId": {
+          "title": "Decision Proposition ID",
+          "type": "string",
+          "description": "An ID associated with an individual decision response. A single decision can produce an array of propositions, one for each activity involved. It is the expectation that applications reference this ID in every experience event that was influended by the propositions. A user profile's proposition-response history is maintained with these identifiers."
+        },
+        "https://ns.adobe.com/experience/decisioning/propositionDetails": {
+          "title": "Propopostion Array providing the outcomes of a single decision event, one outcome per given activity",
+          "type": "array",
+          "items": {
+            "$ref": "https://ns.adobe.com/experience/decisioning/proposition#/definitions/proposition-detail-datatype"
+          },
+          "description": "Details about the individual context variables and outcomes for the decision."
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/proposition-detail-datatype"
+    },
+    {
+      "required": ["xdm:activity"]
```

Comment:
I don't think this is a good idea to require this as it will become always required which will impact unioning these ExperienceEvents with ones from other sources..

Reply:
Please note that the property xdm:activity is a datatype property and not a mixin property. 
For mixins we have the properties `https://ns.adobe.com/experience/decisioning/propositionDetails` and `https://ns.adobe.com/experience/decisioning/propositionId` 
Neither of them are marked `required`. The first one is an object data type and this is where the property `xdm:activity` is showing up and it is also within that datatype where its presence is required. 
If and only if the mixin `https://ns.adobe.com/experience/decisioning/propositionDetails` is present then the JSON list structure has additional requirements. The list is a list of objects and the objects must have at a minimum the xdm:activity property.

So it does not affect the unioning of ExperienceEvents after all